src/main/java/com/zerozae/voucher/controller/api/ApiCustomerController.java

```diff
@@ -0,0 +1,68 @@
+package com.zerozae.voucher.controller.api;
+
+import com.zerozae.voucher.dto.customer.CustomerCreateRequest;
+import com.zerozae.voucher.dto.customer.CustomerUpdateRequest;
+import com.zerozae.voucher.service.customer.CustomerService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+import static com.zerozae.voucher.validator.InputValidator.validateInputUuid;
+import static org.springframework.http.HttpStatus.CREATED;
+import static org.springframework.http.HttpStatus.OK;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/customers")
```

Comment:
api에 대한 버전도 명시하면 좋습니다
```suggestion
@RequestMapping("/api/v1/customers")
```

Reply:
https://github.com/prgrms-be-devcourse/springboot-basic/pull/945/commits/8895499ecf29864792bcc6c2d95bbbab9aad3d3c